starport/cmd/relayer_configure.go

```diff
@@ -328,7 +328,16 @@ func relayerConfigureHandler(cmd *cobra.Command, args []string) error {
 
 	s.Stop()
 
+	info, err := xrelayer.Info(cmd.Context())
+	if err != nil {
+		return err
+	}
+
 	fmt.Printf("⛓  Configured chains: %s\n\n", color.Green.Sprint(path.ID))
+	fmt.Printf(`Warning!
```

Comment:
Here.

Reply:
```
Note: mnemonics for relayer accounts are stored in %s unencrypted.
This may change in the future. Until then, use them only for small amounts of tokens.
```

Reply:
I don't want to use "Warning", because it sounds like a user has done something wrong or dangerous.

Reply:
solved by b113443.